OpenRA.Game/GameInformation.cs

```diff
@@ -114,7 +114,7 @@ public void AddPlayer(OpenRA.Player runtimePlayer, Session lobbyInfo)
 				Color = runtimePlayer.Color,
 				Team = client.Team,
 				SpawnPoint = runtimePlayer.SpawnPoint,
-				IsRandomFaction = runtimePlayer.Country.Race != client.Country,
+				IsRandomFaction = runtimePlayer.Country.Race != client.CountryRace,
```

Comment:
It was used in the same manner as `Race` but I figured someone would get tripped up by that so I just appended `Race` to the field instead.

`Country.Race != Country` is a bit bogus IMO.

It's the best I could do in a bad situation.